rpcs3/Loader/TRP.cpp

```diff
@@ -50,7 +50,7 @@ bool TRPLoader::Install(const std::string& dest, bool show)
 
 	if (success)
 	{
-		success = vfs::host::remove_all(local_path, Emu.GetHddDir(), true);
+		success = vfs::host::remove_all(local_path, Emu.GetHddDir(), true) || !fs::is_dir(local_path);
```

Comment:
Not relaying on ENOENT here as this error may bubble up from some inside operations like file deletions etc.